### PR TITLE
I have completed html-tables exercise.

### PR DESCRIPTION
**Pull request by AngelaJiaNan**
Screen Shot 2021-10-05 at 2 27 23 PM

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head> <title>HTML table</title>
+</head>
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+    <tr>
```
Comment (Zhiwutian) [inline html-tables/index.html:8]: just make sure to add the `thead` element for the first table row.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looks great! Just a tiny update to add the `thead` element

File: html-tables/index.html (hunk 1 above)
Comment (AngelaJiaNan) [reply html-tables/index.html:9]: added it in and pushed though

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**